app/views/schools/choose_programme/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+  <h1 class="govuk-heading-xl">Welcome</h1>
+  <%= form_for @induction_choice_form, url: schools_choose_programme_path, method: :post do |f| %>
+    <%= f.govuk_collection_radio_buttons(
+          :programme_choice, @induction_choice_form.programme_choices, :id, :name,
+          legend: { text: "How do you want to run your induction in the 2021/22 academic year?", size: "m" },
```

Comment:
I prefer one argument per line, it's a bit hard to red when you have to guess how many arguments there are per line (or look for commas)

Reply:
Agreed, this was mostly moved, not written from scratch